README.md

```diff
@@ -1,3 +1,5 @@
+<img width="110" src="https://user-images.githubusercontent.com/9271177/36511315-150065a8-175e-11e8-8f7a-27d3f72aa52b.png" />
```

Comment:
Where is the image hosted?

Reply:
The image is stored on some GitHub server, using [this](https://www.youtube.com/watch?v=wVHJtL-y7P0) approach. I saw that some repos are using this (eg: [here](https://raw.githubusercontent.com/zeit/next.js/canary/readme.md), [here](https://raw.githubusercontent.com/react-boilerplate/react-boilerplate/master/README.md)). Thought, i can´t find anything related to expiry policy.